src/Model/GroundWaterTransport/gwt1cnc1.f90

```diff
@@ -199,79 +217,69 @@ subroutine cnc_ad(this)
     !    "current" value.
     call this%obs%obs_ad()
     !
-    ! -- return
+    ! -- Return
     return
   end subroutine cnc_ad
 
+  !> @brief Check constant concentration/temperature boundary condition data
+  !<
   subroutine cnc_ck(this)
-! ******************************************************************************
-! cnc_ck -- Check cnc boundary condition data
-! ******************************************************************************
-!
-!    SPECIFICATIONS:
-! ------------------------------------------------------------------------------
     ! -- modules
-    use ConstantsModule, only: LINELENGTH
-    use SimModule, only: store_error, count_errors, store_error_unit
     ! -- dummy
     class(GwtCncType), intent(inout) :: this
     ! -- local
-    character(len=LINELENGTH) :: errmsg
     character(len=30) :: nodestr
     integer(I4B) :: i
     integer(I4B) :: node
     ! -- formats
     character(len=*), parameter :: fmtcncerr = &
-      &"('CNC boundary ',i0,' conc (',g0,') is less than zero for cell', a)"
-! ------------------------------------------------------------------------------
+      &"('Specified dependent variable boundary ',i0, &
+      &' conc (',g0,') is less than zero for cell', a)"
     !
     ! -- check stress period data
     do i = 1, this%nbound
       node = this%nodelist(i)
       ! -- accumulate errors
-      if (this%bound(1, i) < DZERO) then
+      if (this%tspvar(i) < DZERO) then
```

Comment:
I think instead of tspvar, you should be using conc_mult() everywhere instead, but you should verify with @mjreno.

Reply:
Thanks @langevin-usgs, that is right.  As conc_mult() was added it should replace every other use of tspvar where the scaled value should be used- and we should use this same pattern broadly for all other package files that define an analogous *_mult() function.

Reply:
addressed with a12332c